src/cpp/lib/testml/wrapper.cpp

```diff
@@ -0,0 +1,29 @@
+#include "wrapper.hpp"
```

Comment:
It's not a separate class. These functions will supposedly be reused by the Bridge"-r" and the Stdlib class, that's why it's separated into its own file.

Reply:
I'm not certain that the bridge classes will (or should) know about and use testml runtime internals, but we can wait and see.

Reply:
I don't understand.

Reply:
@vendethiel now that we forged a common understanding on IRC, I get where you are going.

In TestML lingo a 'method' call like `.foo` or `.Bar` is called a 'callable'. `.foo` is a user defined bridge class callable (because it is lower case) and `.Bar` is a stdlib callable. Given that the `testml::Bridge` (base class for user defined `TestMLBridge` class) and `testml::StdLib` both need the wrapper stuff I think that wrapper should be a `testml::Callable`  class and both `testml::Bridge` and `testml::StdLib` should inherit from it.